documentation/src/test/java/org/hibernate/userguide/mapping/basic/BitSetUserType.java

```diff
@@ -21,7 +21,7 @@
 //tag::basic-custom-type-BitSetUserType-example[]
 public class BitSetUserType implements UserType {
 
-	public static final BitSetUserType INSTANCE = new BitSetUserType();
+    public static final BitSetUserType INSTANCE = new BitSetUserType();
 
     private static final Logger log = Logger.getLogger( BitSetUserType.class );
 
```

Comment:
changing the TAB to 4 space chars fixed the inconsistent indentation as demonstrated below
<img width="1230" alt="Screen Shot 2021-11-17 at 02 00 25" src="https://user-images.githubusercontent.com/8211458/142149883-2659eb7e-bad6-40ce-8d69-8bba7d077be7.png">
:

It might be an idea to substitute space chars for the TAB chars in code snippets in user guide to fix the 'cross-code' inconsistency issue.